datadistillr/datadistillr.py

```diff
@@ -43,7 +43,7 @@ def get_dataframe(url, api_key):
     @staticmethod
     def make_api_call(url, api_key):
         headers = {"Authorization": api_key}
-        if "devapp.datadistillr" in url:
+        if "app.datadistillr" in url:
```

Comment:
@Amandaha29-ddr This piece was here because the dev instance was throwing certificate errors.  We can either remove the `if` statement entirely and keep
```
requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
response = requests.get(url, headers=headers, verify=False)
```
OR just 

```
response = requests.get(url, headers=headers)
```

The latter is more secure.  If that works w/o errors I'd say we do that.